instructor/function_calls.py

```diff
@@ -63,11 +65,7 @@ def openai_schema(cls) -> Dict[str, Any]:
     @property
     def anthropic_schema(cls) -> str:
         from instructor.anthropic_utils import json_to_xml
-
-        return "\n".join(
-            line.lstrip()
-            for line in parseString(json_to_xml(cls)).toprettyxml().splitlines()[1:]
-        )
+        return json_to_xml(cls)
```

Comment:
Breaks with List[x] as it inserts non-XML into the result. I'll add back the prompt clean up but this fixes current critical issue in main branch.